keyboards/keebio/iris/keymaps/osiris/readme.md

```diff
@@ -0,0 +1,9 @@
+# My Iris Layout
+
+![My Iris Rev3](https://imgur.com/a/FgcAPeZ)
```

Comment:
This line should be linked directly to your image.

```suggestion
![My Iris Rev3](https://i.imgur.com/7oXacel.jpg)
```

Reply:
updated link